### PR TITLE
library upgrade, changelog, gitignore

### PR DESCRIPTION
**Pull request by CityVibes**
upgrading android support, cloud sdk and setup libraries

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -49,8 +49,8 @@ android {
 dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
 
-    compile 'io.particle:cloudsdk:0.3.4'
-    compile 'io.particle:devicesetup:0.3.5'
+    compile 'io.particle:cloudsdk:0.4.0'
```
Comment (jensck) [inline app/build.gradle]: 👍

File: app/build.gradle [hunk 2]
```diff
@@ -65,11 +65,11 @@ dependencies {
     compile 'com.squareup.phrase:phrase:1.0.3'
     compile 'uk.co.chrisjenx:calligraphy:2.2.0'
 
-    compile 'com.android.support:appcompat-v7:25.1.0'
-    compile 'com.android.support:design:25.1.0'
-    compile 'com.android.support:recyclerview-v7:25.1.0'
-    compile 'com.android.support:support-fragment:25.1.0'
-    compile 'com.android.support:support-annotations:25.1.0'
+    compile 'com.android.support:appcompat-v7:25.1.1'
```
Comment (jensck) [inline app/build.gradle]: 👍

File: certificate.cer [hunk 3]
```diff
@@ -0,0 +1,42 @@
+-----BEGIN CERTIFICATE-----
```
Comment (jensck) [inline certificate.cer]: Er, what's this?

Comment (CityVibes) [inline certificate.cer]: Oh sorry, did include some stuff from my testing. Removed all those files.

**Comment by jensck** [comment]
I'm going to do a squash merge on this to prevent the accidentally committed binaries from staying in git history forever.

**Pull request merged**